src/settings/toml/tests/mod.rs

```diff
@@ -147,5 +147,5 @@ fn base_fixture_path() -> PathBuf {
 }
 
 fn toml_fixture_path(fixture: &str) -> PathBuf {
-    base_fixture_path().join(fixture)
+    base_fixture_path().join(fixture).with_extension("toml")
```

Comment:
why is this change necessary?

Reply:
some magical thing in `config` crate was doing this for us when we went to use `merge`, but now that we first check `config_path.exists()` before we get to that point, the extension needs to be explicitly included in the path.